### DISCUSSION TITLE
Cannot export unpushed codespace changes to a branch

### DISCUSSION THREAD

**Discussion by eliaselliotson** [community/community#145515] [Codespaces] [answered]
Edit:
I contacted GitHub support, and they got back to me within 10 hours.  They were very kind, and helped me quickly recover my changes.  If you're having the same problem, I'd recommend contacting them as well.

### Select Topic Area

Show & Tell

### Body

I am working on a software project for my church using Github Codespaces.  However, I've used my 120 monthly hours so I can't use the codespace until the month ends.  I've tried exporting my changes to a branch, but it usually loads for a while before failing.  I've tried downloading the codespace using the GitHub cli, but that tells me that I can't access the codespace because my hours are up.  I've checked for other issues and none of them have proper solutions.  Can any of you guys suggest a solution to this?

**Comment by adoubt** [comment]
same 🚬

**Comment by DevNatrix** [comment, marked as answer]
you can contact GitHub support and explain the situation. They might be willing to temporarily increase your hours for the month, especially if it's for a non-commercial, church-related project.

**Reply by eliaselliotson** [reply]
Thank you so much for the quick reply!  I think I might just do that.

**Comment by skripchak2** [comment]
Мне поддержка не помогла, прийдёться ждать месяц.

**Reply by adoubt** [reply]
А мне дали сабку на месяц
